src/data/real/ennreal.lean

```diff
@@ -1210,9 +1210,18 @@ end
 lemma inv_two_add_inv_two : (2:ℝ≥0∞)⁻¹ + 2⁻¹ = 1 :=
```

Comment:
Is it worth adding `inv_three_add_inv_three_add_inv_three` for consistency?

Reply:
Done!